metricbeat/helper/prometheus/metric.go

```diff
@@ -57,6 +57,13 @@ func OpLowercaseValue() MetricOption {
 	return opLowercaseValue{}
 }
 
+// OpMultiplyBuckets multiplies bucket labels in histograms, useful to change units
+func OpMultiplyBuckets(multiplier float64) MetricOption {
```

Comment:
I'd call this `NewOpMultiplyBucketsOption` unless we have some internal convention for this

Reply:
I don't have an strong opinion on this but yes, I am following the name schemas used here.

Take into account that they are not so bad as they are used as options in prometheus metrics:
```
Metrics: map[string]MetricMap{
    "histogram_decimal_metric": Metric("histogram.metric", OpMultiplyBuckets(1000))
}
```